Development-Guide/Vim-Astyle-Guide/README.adoc

```diff
@@ -49,6 +51,16 @@ Now at any time running this command in neovim will format your code into Astyle
 :Autoformat
 ----
 
+To use Autoformat with our astylerc we need to add the code
+
+[source,c++]
+----
+let b:formatdef_custom_c='"astyle --mode=c --suffix=none --options=/file/path/to/astylerc"'
+let b:formatters_c = ['custom_c']
+----
+
+Change this file path to be to where your astylerc is saved to.
```

Comment:
Maybe this should change to become:

1) In your local files, navigate to: Recruit-Resources/Development-Guide/Style-Guide/
2) ``$echo  `pwd` /astylerc`` to see what the `/file/path/to/astylerc/` should be